package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "3.27.0",
+  "version": "3.29.0",
```

Comment:
Anticipating that https://github.com/department-of-veterans-affairs/vets-json-schema/pull/190 will be merged before this.